packages/circuits/helpers/rsa.circom

```diff
@@ -92,7 +92,7 @@ template RSAPad(n, k) {
             if (i+8 < n*k) {
                 modulus_prefix += modulus_bits[i+8];
                 if (i % 8 == 0) {
-                    var idx = (i - (base_len + 8)) / 8;
+                    var idx = (i - (base_len + 8)) \ 8;
```

Comment:
these are the same no, since i and base_len % 8 == 0?

Reply:
Yes, since base_len is divisible by 8.

From the audit report
```
In this section of the code, the division operator / is a modulo division, which is not suitable for the numerical division required by the context. Although (i - (base_len + 8)) happens to be divisible by 8 , using / operator here can be risky as it may produce unexpected results if the operands are not guaranteed to be divisible. 
```

I don't see any benefit for this change tbh, mostly to address the audit finding.